mathesar/serializers.py

```diff
@@ -4,22 +4,17 @@
 from mathesar.models import Table, Schema
 
 
-class ColumnSerializer(serializers.Serializer):
-    name = serializers.CharField()
-    type = serializers.CharField()
-
-
-class RecordSerializer(serializers.BaseSerializer):
-    def to_representation(self, instance):
-        return instance._asdict()
-
-
 class SchemaSerializer(serializers.HyperlinkedModelSerializer):
     class Meta:
         model = Schema
         fields = ['id', 'name', 'database', 'tables']
 
 
+class ColumnSerializer(serializers.Serializer):
+    name = serializers.CharField()
+    type = serializers.CharField()
```

Comment:
Will we also need to show other attributes (I'm thinking `fkey` should be slightly different from type, we might also want to note `primary_key` columns and whether they're `nullable`)

Reply:
Yeah, I think we'll want to show those later. I didn't make any changes to this code in this PR, I just moved it around.